packages/playwright-core/src/server/bidi/third_party/bidiProtocolCore.ts

```diff
@@ -939,7 +939,9 @@ export namespace BrowsingContext {
     defaultValue?: string;
   };
 }
-export type EmulationCommand = Emulation.SetGeolocationOverride;
+export type EmulationCommand =
```

Comment:
There are already more commands in the [upstream version](https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/src/protocol/generated/webdriver-bidi.ts) of the file, can you just copy the one from https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/src/protocol/generated/webdriver-bidi.ts with all the recent changes?

Reply:
Done